source/manual/data-sources-for-transition.html.md

```diff
@@ -0,0 +1,26 @@
+---
+owner_slack: '#navigation'
+review_by: 2017-07-26
+title: Data Sources for Transition
+section: Transition
+layout: manual_layout
+parent: "/manual.html"
+old_path_in_opsmanual: "../opsmanual/2nd-line/applications/bouncer-and-transition.html"
+---
+
+# Data sources for Transition
+
+Site configuration is automatically imported on an hourly basis via
+[deploy.publishing](https://deploy.publishing.service.gov.uk/job/Transition_load_site_config/) from
+[transition-config](https://github.com/alphagov/transition-config).
+
+Pre-transition traffic data is imported from
+[pre-transition-stats](https://github.com/alphagov/pre-transition-stats),
+based on logs provided by transitioning organisations. This was updated
```

Comment:
Has the whole thing (importing traffic data) come to an end, or just the updating by hand? If this is historical, suggest rewriting the whole paragraph so it's in the past tense.